llvm/lib/ProfileData/InstrProfReader.cpp

```diff
@@ -571,9 +559,20 @@ Error RawInstrProfReader<IntPtrT>::readHeader(
   if (!useCorrelate() && Correlator)
     return error(instrprof_error::unexpected_debug_info_for_correlation);
 
-  BinaryIdsSize = swap(Header.BinaryIdsSize);
-  if (BinaryIdsSize % sizeof(uint64_t))
+  uint64_t BinaryIdsSize = swap(Header.BinaryIdsSize);
+  // Binary ids start just after the header.
+  const uint8_t *BinaryIdsStart =
+      reinterpret_cast<const uint8_t *>(&Header) + sizeof(RawInstrProf::Header);
+  const uint8_t *BinaryIdEnd = BinaryIdsStart + BinaryIdsSize;
```

Comment:
Can we call it `BinaryIdsEnd` for consistency with `BinaryIdsStart` and `BinaryIdsSize`?

Reply:
Renamed `BinaryIdsStart` and `BinaryIdsSize` to `BinaryIdStart` and `BinaryIdSize` because there can only be up to one binary id after the header.